README.md

```diff
@@ -430,14 +430,16 @@ In order to have the proxy running, there are several basic pieces of informatio
     port: 10026
 },
 ```
-* `config.access`: connection information to the selected Access Control PDP API. Includes a `disable` flag, to allow the proxy to work in authentication-only mode. E.g.:
+* `config.access`: connection information to the selected Access Control PDP API. Includes a `disable` flag, to allow the proxy to work in authentication-only mode and a flag to activate accounting access in file. E.g.:
```

Comment:
The format of the accountinf file (along with some example) should be included as part of the documentation. Not besides this configuration description, but probably near the parts where log files are described.

Reply:
Fixed in 3 commits on September 25th, 2017.